### ISSUE TITLE
Integer is not string 

### ISSUE THREAD

**Issue by rtfpessoa** [twilio/guardrail#342]
In a spec with a query parameter that is type string, if the client sends a string that is a number it fails to read the value.

e.g.:
```
        - in: path
          name: remoteUserId
          description: Remote user identifier
          required: true
          type: string
          x-example: 1
```

A solution would be to put `"` in the parameter when sending or in akka server being able to convert the int to string which should not be a problem.

Any ideas here?

**Comment by blast-hardcheese** [comment]
@rtfpessoa This should be resolved in #292, can you try to `./cli.sh` on master and verify that this is resolved for you?

**Comment by rtfpessoa** [comment]
Seems like it is fixed :+1:
